deliveries/pallet_supersig_milestone-1_decentration.md

```diff
@@ -0,0 +1,49 @@
+
+**Supersig link**
+https://github.com/w3f/Grants-Program/blob/master/applications/pallet_supersig.md
+
+
+* **Milestone Number:** 1, and half of 3. 
+
+**Context** 
+
+Supersig is like multisig but with superpowers. This means that you can add and remove members from the supersig through voting via simpleMajority, instead of fixed thresholds. 
+
+Secondly, there is also the "master" user feature. A master user always has 50% of the voting power. This means that if a master user is involved in a transaction, only one other person needs to sign in order to approve the transaction for execution. (More than one person can be a master user). 
+
+Supersig is now part of Kabocha parachain's main net on Kusama. 
+
+Milestone 1 is about building pallet_supersig's first version. The pallet now allows you to do the abovementioned core functions. 
+
+We have also apportioned half of milestone 3's feedback and imrprovements, section to: 
+a) create RPC module to make it much easier to query the storage.  
+b) change from Account nonce to AccountId to for usability. 
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+> 
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+| Number | Deliverable | Link | Notes |
+| -----: | ----------- | ------------- | ---------- | 
+| 0a. | License | Apache 2.0  |  | |
+| 0b. | Documentation | https://github.com/decentration/substrate-supersig-template/tree/v0.9.28 | |
+| 0c. | Testing Guide | https://github.com/kabocha-network/pallet_supersig/tree/polkadot-v0.9.28/src/tests | cargo test `INSERT FILENAME` |
+| 0d. | Docker |  https://github.com/decentration/substrate-supersig-template/tree/v0.9.28 | |
+| 0e. | Article and video | https://decentration.medium.com/supersig-like-multisig-but-with-superpowers-86b9ce0412f6, https://www.loom.com/share/dbcaa6319b1a4644aacb709aa0e38783 | |
+| 1. | Substrate module: pallet_supersig | https://github.com/kabocha-network/pallet_supersig/tree/polkadot-v0.9.28/ | |
+| 2. | Benchmarking | The pallet is benchmarked. |https://github.com/kabocha-network/pallet_supersig/blob/polkadot-v0.9.28/src/benchmarking.rs | |
+
+**Additional Information**
+> Any further comments on the milestone that you would like to share with us.
+
+This milestone was completed, we also used half of milestone 3 (Feedback and improvements), to iterate supersig and add RPC modules to make it much more usable. The other half of milestone 3 will be used for milestone 2 (UI development).
```

Comment:
You're supposed to create 1 delivery document for each delivery. Hence, please update this doc to reflect the changes for only M1.

Reply:
hi @aminubi, i just pushed the updated branch to the grant milestone delivery which is now this branch: https://github.com/decentration/substrate-supersig-template/tree/v0.9.28-new, which compiles and runs.

Reply:
@decentration another Error: Service(Keystore(Io(Os { code: 13, kind: PermissionDenied, message: "Permission denied" }))) when running with docker image.

Reply:
@aminubi try this `docker run -p 9944:9944 -p 30333:30333 -p 9933:9933 -p 9615:9615  docker.io/decentration/supersig-node:latest --dev --port 30333 --ws-port 9944 --rpc-port 9933`. I have updated the readme to contain this command. It should start a local node template, and publish to the correct ports so you can interact with it from polkadot js apps.

Reply:
though if you are on macOS you may get some problems when connecting via polkadot js apps UI. Trying to get to the bottom of it, but it seems its a wider Docker MacOS issue.

Reply:
here is the issue from [docker](https://github.com/docker/for-mac/issues/3487#issuecomment-509176608). But i can confirm it works for Mac Intel chip.

Reply:
hey @aminubi, how is the docker running?

Reply:
I'll mark this as resolved, as the initial request has been changed, and the docker is also working. If you still have an issue please let me know.